src/component/Categories.jsx

```diff
@@ -1,17 +1,14 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
-import { getStatus } from '../redux/categories/categoriesSlice';
+// import React from 'react';
+// import { useSelector } from 'react-redux';
 
-function CategoryList() {
-  const categories = useSelector(getStatus);
-  const handleClick = () => `Status: ${categories}`;
+// function CategoryList() {
+//   const categories = useSelector();
+//   return (
+//     <div>
+//       <button type="button">Get Status</button>
+//       <div />
+//     </div>
+//   );
+// }
 
-  return (
-    <div>
-      <button type="button" onClick={handleClick}>Get Status</button>
-      <div />
-    </div>
-  );
-}
-
-export default CategoryList;
+// export default CategoryList;
```

Comment:
- [ ] @Kwamay It's not good practise to have commented code in your codebase, kindly consider resolving errors in your categories component instead of commenting on the whole file.